src/nvim/os/shell.c

```diff
@@ -68,6 +68,18 @@ char **shell_build_argv(const char *cmd, const char *extra_args)
   return rv;
 }
 
+char **build_argv(char *cmd)
```

Comment:
Those examples would be great to have in a `///` comment above each respective function.